### DISCUSSION TITLE
Cording

### DISCUSSION THREAD

**Discussion by deadsec76** [community/community#40238] [Mobile]
<details>Osintgram
<summary>ℹ️ Additional Debug Info</summary>lil baby
GitHub for iOS, Version 1.90.0 (84076705)
Device: iPhone 6S Plus (iOS 15.7.1)
</details>100

**Comment by Akash1134** [comment]
Hi @deadsec76  , we're happy you're here! You are more likely to get a useful response if you are explicit about what your project entails, giving a few more details might help someone give you a nudge in the right direction.

**Comment by deadsec76** [comment]
> <details>
> <summary>ℹ️ Additional Debug Info</summary>
> GitHub for iOS, Version 1.90.0 (84076705)
> Device: iPhone 6S Plus (iOS 15.7.1)
> </details>
